Content.Shared/GameObjects/Components/Weapons/Melee/MeleeWeaponAnimationPrototype.cs

```diff
@@ -10,16 +11,16 @@ namespace Content.Shared.GameObjects.Components.Weapons.Melee
     [Prototype("MeleeWeaponAnimation")]
     public sealed class MeleeWeaponAnimationPrototype : IPrototype, IIndexedPrototype
     {
-        private string _prototype;
-        private string _state;
-        private string _id;
+        private string _prototype = "WeaponArt";
```

Comment:
Shouldn't be this as A) it gets overridden and B) it is misspelt.

Reply:
Fixed spelling, kept default value as discussed in Discord.